Extensions/TextInput/textinputruntimeobject-pixi-renderer.ts

```diff
@@ -119,6 +119,8 @@ namespace gdjs {
       if (!this._input) return;
       this._input.remove();
       this._input = null;
+      this._form?.remove();
```

Comment:
nitpicking: either use `?.` for the input too or use `!this._form`, but be consistent between the twos.